pkg/image/api/helper.go

```diff
@@ -168,6 +169,17 @@ func (r DockerImageReference) RepositoryName() string {
 	return r.Exact()
 }
 
+// RegistryHostname returns the registry hostname without port
+func (r DockerImageReference) RegistryHostPort() (string, string) {
```

Comment:
Shouldn't this method accept bool `Insecure`? And return `80` if it's set to `true`?